stumpy/core.py

```diff
@@ -679,15 +706,30 @@ def rolling_nanstd(a, w):
     w : numpy.ndarray
         The rolling window size
 
+    welford : bool, default False
+        When False (default), the computation is parallelized and the std of
+        each subsequence is calculated on its own. When `welford==True`, the
+        welford method is used to reduce the computing time at the cost of slight
+        loss of precision
+
     Returns
     -------
-    output : numpy.ndarray
-        Rolling window nanstd.
+    out : numpy.ndarray
+        Rolling window nanstd. When `a` is 1D, `out[i]` is the std of `a[i, i + w]`.
```

Comment:
Rolling window nanstd

Reply:
Did you mean I should remove the rest of description for `out`?

Reply:
Correct. I meant that you should simply shorten it to what I suggested above and remove everything else